### DISCUSSION TITLE
Incorrect link for `View today's article`

### DISCUSSION THREAD

**Discussion by aero-girl** [AzureAiDevs/hub#10] [azure-ai-developers]
Hey 👋🏽,

I have just come across #30DaysOfAzureAI and was going through the [1. 🚀Kick Start AI April Keynote](https://azureaidevs.github.io/hub/2023-aia/day1). Thank you for putting this together but thought I'd highlight that the link to read today's article takes you to the Global AI Dev Conf Keynote video.
![image](https://user-images.githubusercontent.com/5180795/228273170-335c0124-6948-461d-a075-182523b1d94b.png)

Apologies if this is the wrong place to post this!

**Comment by gloveboxes** [comment]
Hi there @aero-girl, thank you for pointing out, I've removed the Today's link as it just pointed to the same YT shared on the page and I can see that doesn't make sense. Thanks again! Cheers Dave
